aws/iam/corp.tf

```diff
@@ -4,6 +4,32 @@ variable "corp_account_user_names" {
   default     = []
 }
 
```

Comment:
Not sure that this is the correct logical grouping. While *yes*, it's IAM related, it's highly subjective to an accounts implementation. 

Feels like the rock-solid place to stick it that's always true is with `kops` or `kops-aws-platform`

Reply:
(That will also be more dry)